Formula/coinutils.rb

```diff
@@ -0,0 +1,50 @@
+class Coinutils < Formula
+  desc "COIN-OR utilities"
+  homepage "https://github.com/coin-or/CoinUtils"
+  url "https://github.com/coin-or/CoinUtils/archive/releases/2.11.2.tar.gz"
+  sha256 "30c7f6c84dbb9f6e4fe5bbe4015ed15e2d1402146f8354cfc50c34d8735a49b1"
+
+  depends_on "pkg-config" => :build
+  depends_on "gcc"
```

Comment:
Why does this need GCC? It's a very heavy dependency, and we prefer to build formulas with the system compiler.

Reply:
It depends on fortran (`gfortran`): https://github.com/coin-or/CoinUtils/blob/master/CoinUtils/configure.ac#L52-L59

Reply:
There is not a single line of Fortran in the project, it should be able to compile without it.

Reply:
Removed